### ISSUE TITLE
Create a stat page for admin

### ISSUE THREAD

**Issue by NicholasChoong** [codersforcauses/poops#147]
**Is your feature request related to a problem? Please describe.**
Admins need a page to see how their users are doing.

**Describe the solution you'd like**
Create a stats page for the admin based on the design made in the figma #135 .
The page should consist of:
- Per volunteer:
  - Total # visits, minutes, kms over the period
  - Average number of visits, minutes, kms per week
  - Average number of clients

**Additional context**
You dont have to worry about Firebase integration or restricting to access to this page.
This will be created in another issue.

**Comment by scratchclaggy** [comment]
Note, this is blocked by #135
